Add to product_package domain product.product model.

juanjo-nan opened the pull request:
Task #067415

juanjo-nan committed:
Add to product_package domain product.product model.

Task #067415

juanjo-nan committed:
Check if we have default packages in product.product and product.template at the same time.

Task #067415

bernatnan commented on purchase.py:
```diff
@@ -37,7 +39,7 @@ class PurchaseLine(metaclass=PoolMeta):
             'readonly': Eval('purchase_state') != 'draft',
             },
-        depends=['product_has_packages', 'purchase_state'])
+        depends=['product_has_packages', 'purchase_state', 'proudct'])
```
Aquest "product" va al depends del product_package.

bernatnan commented on purchase.py:
```diff
@@ -69,11 +71,19 @@ def on_change_product_supplier(self):
+                        break
 
     @fields.depends('product')
     def on_change_with_product_has_packages(self, name=None):
```
Aquí no faltaria comprovar els 2 llocs, com s'ha fet en el sale?

if self.product and (self.product.packages
        or self.product.template.packages):

juanjo-nan replied:
Ja s'està comprovant pel model product.product i product.template:
https://github.com/NaN-tic/trytond-purchase_product_package/blob/c7e169cc76f6384409ecb68042e090586d6a8ed6/purchase.py#L70-L86

Pot ser estaves mirant el codi sense modificar?

juanjo-nan committed:
FIX depends.

Task #067415

juanjo-nan committed:
FIX domain.
Add missing check template in "on_change_with_product_has_packages".

Task #067415
